File: src/Services/Identity/Identity.Common/Models/Student.cs
using System;
using System.Collections.Generic;

namespace Identity.Common.Models
{
    public class Student : User
    {

        public DateTime DateOfBirth { get; set; }
        public string Nationality { get; set; }
        public string StreetName { get; set; }
        public int? HouseNumber { get; set; }
        public string PostalCode { get; set; }
        public string City { get; set; }
        public string Country { get; set; }

        public string ProgramId { get; set; }
        public Program Program { get; set; }

        public List<Mentor> Mentors { get; set; }

    }
}

File: src/Web/Spa/src/app/models/student.ts
import { Program } from './program';
import { Observable } from 'rxjs';

export class Student {
    id: number;  
    pcn: string;
    firstName: string;
    lastName: string;
    initials:string;
    email:string;
    phoneNumber:string;
    accountStatus:string;
    dateOfBirth:Date;
    nationality:string;
    streetName:string;
    houseNumber:number;
    postalCode:string;
    city:string;
    country:string;
    program:Program;
    
  }
